maint/code-cleanup.sh

```diff
@@ -1,6 +1,6 @@
 #! /bin/bash
 
-if test ! -z "`which gindent`" ; then
+if hash gindent 2>/dev/null; then
```

Comment:
Probably way more hairsplitting that needed on this PR, but why this change? If needed, should be 3 commits total, IMO.